libs/full/compute_cuda/examples/hello_compute.cu

```diff
@@ -48,40 +48,37 @@ int hpx_main(hpx::program_options::variables_map& vm)
     hpx::compute::vector<int, allocator_type> d_C(N, alloc);
 
     hpx::parallel::copy(
-        hpx::parallel::execution::seq,
-        h_A.begin(), h_A.end(), d_A.begin());
+        hpx::parallel::execution::seq, h_A.begin(), h_A.end(), d_A.begin());
```

Comment:
Shouldn't this be `hpx::execution::seq`?

Reply:
It should :/ I suppose the flags don't get applied for nvcc... Will check if we can enable them. I'll fix this in any case. Very good catch!

Reply:
Actually they'll most likely trigger warnings. I just didn't test this with CUDA enabled.